pkg/cri/server/image_pull_test.go

```diff
@@ -435,51 +434,3 @@ func TestSnapshotterFromPodSandboxConfig(t *testing.T) {
 		})
 	}
 }
-func TestImageGetLabels(t *testing.T) {
```

Comment:
if we go this way should still unit test getlabels, isSandboxImage, and pin

Reply:
getValues is very simple, keep this function just to make getting labels more uniform (maybe remove this function, I'm keeping it for now).

Added a test for isSandboxImage, the test for pin is in store/image_test.go

Integration test TestContainerdSandboxImage also cover this part